pkg/object/objmetadata.go

```diff
@@ -29,6 +29,19 @@ import (
 // resource name.
 const fieldSeparator = "_"
 
+type ObjLoader interface {
```

Comment:
Is it useful to split this into so many separate interfaces? I thought it might be easier to understand if we had it all in the Inventory interface.

Reply:
The `go` language does one function interfaces quite often, composing the smaller interfaces into larger interfaces (e.g. the `io` stuff with `Reader`, `Writer`, `Closer`, and `ReadWriter`, `ReadWriteCloser`). But I see your point that maybe this belongs in the `inventory.go`.

Reply:
Fixed